cookbook/showing-authentication-view/README.md

```diff
@@ -0,0 +1,71 @@
+# Showing Authentication View
```

Comment:
Maybe we can think of some better title. How about "Requesting a user to authenticate"?